jurity/recommenders/__init__.py

```diff
@@ -3,19 +3,29 @@
 from .auc import AUC
 from .combined import CombinedMetrics
 from .ctr import CTR
+from .ips import IPS
+from .doubly_robust import DoublyRobust
 from .map import MAP
 from .ndcg import NDCG
 from .precision import Precision
+from .rank_estimation import RankEstimation
 from .recall import Recall
 
 
 class BinaryRecoMetrics(NamedTuple):
     AUC = AUC
     CTR = CTR
+    DoublyRobust = DoublyRobust
+    IPS = IPS
 
 
 class RankingRecoMetrics(NamedTuple):
     MAP = MAP
     NDCG = NDCG
     Precision = Precision
     Recall = Recall
+
+
+class RankingUtils(NamedTuple):
+    RankEstimation = RankEstimation
```

Comment:
Not PR comment, but I am probably missing some knowledge here. 

What's the new RankEstimation class? and why does effect the existing Map/Recall functionality?

Reply:
This is for use cases with negative sampling. The idea is that if the test data is sampled, the rankings in the predictions may not reflect what they would be if all items were included. Something that is in the top k in the experiment may not be in the top k in the real world experiment, it might be a few places lower. Unbiased rank estimation provides a correction to any metrics that rely on ranking (basically everything except AUC which is robust against this effect) to achieve a closer approximation of the ground truth metric.

Reply:
I see.. thanks for the explanation!

Reply:
Yeah the recommendation is to avoid sampling the test data if at all possible, but in cases where you can't avoid it, this will give a better approximation of model performance